tests/gdx-tests/src/com/badlogic/gdx/tests/TiledMapObjectLoadingTest.java

```diff
@@ -79,13 +82,33 @@ public void render () {
 		shapeRenderer.setColor(Color.BLUE);
 		Gdx.gl20.glLineWidth(2);
 		MapLayer layer = map.getLayers().get("Objects");
+		AnimatedTiledMapTile.updateAnimationBaseTime();
 		for (MapObject mapObject : layer.getObjects()) {
-			if (mapObject instanceof TextureMapObject) {
-				TextureMapObject tmObject = (TextureMapObject)mapObject;
+//			if (mapObject instanceof TextureMapObject) {
```

Comment:
I wanted to keep this, because the `TextureMapObject` wasn't removed. It's still there, can be used and also tested like that. However, I could not keep both of them at the same time.